vendor/github.com/xanzy/go-gitlab/client_options.go

```diff
@@ -24,6 +24,16 @@ func WithCustomBackoff(backoff retryablehttp.Backoff) ClientOptionFunc {
 	}
 }
 
+// WithCustomLimiter injects a custom rate limiter to the client.
```

Comment:
I'm not sure if you should be checking in the `go mod vendor` files, if you need changes from the upstream library that aren't in the newest package version you could use the replace directive to point to your own upstream branch.

https://golang.org/ref/mod#go-mod-file-replace

Reply:
It's not my changes but a sync from 0.37.0 (from 0.34.0)